source/fab/language/__init__.py

```diff
@@ -19,7 +19,17 @@ class TaskException(Exception):
 
 class Task(ABC):
     @abstractmethod
-    def run(self) -> List[Path]:
+    def run(self):
```

Comment:
Okay, I wasn't quite sure on that as I've seen it annotated that way in places but thought perhaps we'd decided to leave it out when it is `None`

Reply:
Having thought about it I think I'd like to propose we leave these out when the method doesn't return anything.  Feels like we should be consistent, but I don't think want to start putting them on every single `__init__` and `run` and every other place it would have to go

Reply:
It might be better to be explicit. It adds chatter to the source but it means that if someone converts a function to return something they will get a kick from `mypy` rather than silent aquiescence.

I wouldn't expect you to change all instances of missing `-> None` in this ticket, just the ones you introduce.